package.json

```diff
@@ -77,6 +80,7 @@
     "@commitlint/config-angular": "^17.4.4",
     "@commitlint/config-conventional": "^17.4.4",
     "@readme/eslint-config": "^14.0.0",
+    "@readme/markdown": "^6.84.0",
```

Comment:
peer dependency of itself?

Reply:
this does seem weird; seems like it's just for tests, but not sure why we'd need to actually install it like this instead of just importing it directly?

Reply:
That's acshually a devDep. It's the mainline release, so I can't import it locally.

I was trying to get more confidence that our migration is going to work.

Reply:
you might want to pull it in and re-name it to something other than `@readme/markdown` so npm doesn't think you want to load the raw source instead

Reply:
👍 if it works im cool with that.